k8s/jenkins/legion-jenkins-plugin/src/main/resources/legion.groovy

```diff
@@ -16,14 +16,29 @@
  *   limitations under the License.
  */
 
+class Globals {
```

Comment:
@kirillmakhonin what is the purpose of Globals if the same global keys are overridden in several different lib method? What we will get if these methods are executed in parallel ?  Such global vars should be defined once in some init() method that is executed just once as a first step of pipeline script.